### DISCUSSION TITLE
Q-Input Grow Vertically

### DISCUSSION THREAD

**Discussion by jdscolam** [quasarframework/quasar#15863] [General - Components / Directives / etc] [answered]
Hi there, maybe a stupid/complex question, but I have a situation where there's a q-input affixed to the bottom of a dialog and as the user inputs multiple lines of text I'd like it to grow vertically while staying anchored to the bottom of the screen.  Is this possible?

**Comment by suleimanas** [comment]
Hi, sure it's not stupid nor complex 
by using **autogrow** prop, check this example: 
https://quasar.dev/vue-components/input#example--autogrow

**Reply by jdscolam** [reply]
Great suggestion, unfortunately autogrow is growing down rather than up in my app rather than just up.

**Reply by suleimanas** [reply, marked as answer]
No my friend it doesn't grow up or down it just grows. it's up to your layout
check this out:

```
<div class="column" style="height: 400px">
  <div class="col">
    <div>...</div>
  </div>
  <div>
    <q-input
      v-model="text"
      filled
      autogrow
    ></q-input>
  </div>
</div>
```

**Reply by jdscolam** [reply]
Thank you, that was the perfect answer for my situation.  Have a great day, and again thanks for being such a great help!
